api/mapping/forms.py

```diff
@@ -195,4 +195,13 @@ class ScanReportAssertionForm(forms.Form):
 class NLPForm(forms.Form):
     user_string = forms.CharField(
         label = ""
-    )
\ No newline at end of file
+    )
+
+class ScanReportValueConceptForm(forms.Form):
+    scan_report_value_id=forms.IntegerField(
```

Comment:
Do we want to keep fields with no arguments on a single line?

Reply:
I like this approach because if a parameter is added in the future, it will involve one change, instead of 3 (one substitution and two additions).

Reply:
@py5gol Personally, I don't see what the issue is of 1 change versus 3 (from a Git perspective).
I think the code is more readable if this is on a single line - I consider readabiloty more important than whether Git cosiders it 1 or 3 changes. Just my 2c anyway!